tests/cloudformation/checks/resource/aws/example_SQSQueueEncryption/test_SQSQueueEncryption-FAILED.yml

```diff
@@ -5,4 +5,5 @@ Resources:
     Properties:
       RedrivePolicy:
         deadLetterTargetArn: "example_arn"
-        maxReceiveCount: 5
\ No newline at end of file
+        maxReceiveCount: 5
+      KmsMasterKeyId: ""
```

Comment:
didn't know that this actually a valid configuration 😱

Reply:
Perhaps it's not 🤷‍♀️  I just wanted to validate the string check works for cloudformation too